Reason: Skip tests that require npm on riscv64
Upstream: No

diff --git a/hugolib/js_test.go b/hugolib/js_test.go
index cd0883fd..aeb784fe 100644
--- a/hugolib/js_test.go
+++ b/hugolib/js_test.go
@@ -34,6 +34,10 @@ import (
 )
 
 func TestJSBuildWithNPM(t *testing.T) {
+	if runtime.GOARCH == "riscv64" {
+		t.Skip("npm is not available on riscv64")
+	}
+
 	if !htesting.IsCI() {
 		t.Skip("skip (relative) long running modules test when running locally")
 	}
@@ -142,6 +146,10 @@ function greeter(person) {
 }
 
 func TestJSBuild(t *testing.T) {
+	if runtime.GOARCH == "riscv64" {
+		t.Skip("npm is not available on riscv64")
+	}
+
 	if !htesting.IsCI() {
 		t.Skip("skip (relative) long running modules test when running locally")
 	}
diff --git a/hugolib/resource_chain_babel_test.go b/hugolib/resource_chain_babel_test.go
index 61b2ef6d..d2bb3293 100644
--- a/hugolib/resource_chain_babel_test.go
+++ b/hugolib/resource_chain_babel_test.go
@@ -17,6 +17,7 @@ import (
 	"bytes"
 	"os"
 	"path/filepath"
+	"runtime"
 	"testing"
 
 	"github.com/gohugoio/hugo/common/hexec"
@@ -35,6 +36,10 @@ import (
 )
 
 func TestResourceChainBabel(t *testing.T) {
+	if runtime.GOARCH == "riscv64" {
+		t.Skip("npm is not available on riscv64")
+	}
+
 	if !htesting.IsCI() {
 		t.Skip("skip (relative) long running modules test when running locally")
 	}
diff --git a/hugolib/resource_chain_test.go b/hugolib/resource_chain_test.go
index 9ea1d852..4063a81a 100644
--- a/hugolib/resource_chain_test.go
+++ b/hugolib/resource_chain_test.go
@@ -23,6 +23,7 @@ import (
 	"github.com/gohugoio/hugo/resources/resource_transformers/tocss/dartsass"
 
 	"path/filepath"
+	"runtime"
 	"strings"
 	"testing"
 	"time"
@@ -920,6 +921,10 @@ Hello2: Bonjour
 }
 
 func TestResourceChainPostCSS(t *testing.T) {
+	if runtime.GOARCH == "riscv64" {
+		t.Skip("npm is not available on riscv64")
+	}
+
 	if !htesting.IsCI() {
 		t.Skip("skip (relative) long running modules test when running locally")
 	}
